website/docs/ephemeral-resources/organization_token.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: Ephemeral: tfe_organization_token"
+description: |-
+  Generates a new organization token that is guaranteed not to be written to
+  state.
+---
+
+# Ephemeral: tfe_organization_token
+
+Terraform ephemeral resource for managing a TFE organization token. This
+resource is used to generate a new organization token that is guaranteed not to
+be written to state. Since organization tokens are singleton resources, using this ephemeral resource will replace any existing organization token.
+
+~> **NOTE:** Ephemeral resources are a new feature and may evolve as we continue to explore their most effective uses. [Learn more](https://developer.hashicorp.com/terraform/language/v1.10.x/resources/ephemeral).
+
+## Example Usage
+
+### Generate a new organization token:
+
+This will invalidate any existing organization token.
+
+```hcl
+ephemeral "tfe_organization_token" "example" {
+  organization = "my-org-name"
+}
+```
+
+### Generate a new organization token with 30 day expiration:
+
+This will invalidate any existing organization token.
+
+```hcl
+resource "time_rotating" "example" {
+  rotation_days = 30
+}
+
+ephemeral "tfe_organization_token" "example" {
+  organization   = "my-org-name"
+  expired_at = time_rotating.example.rotation_rfc3339
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `organization` - (Required) Name of the organization. If omitted, organization must be defined in the provider config.
```

Comment:
I think this is optional because it probably can use provider config.

Reply:
Good catch, addressed in 93f913be.